connect/src/com/telenor/connect/AbstractSdkProfile.java

```diff
@@ -3,14 +3,21 @@
 import android.content.Context;
 
 import com.telenor.connect.id.ConnectIdService;
+import com.telenor.connect.id.ConnectTokensTO;
+import com.telenor.connect.utils.IdTokenValidator;
 import com.telenor.connect.utils.RestHelper;
+import com.telenor.connect.utils.Validator;
+
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
 
 import retrofit.Callback;
 import retrofit.RetrofitError;
 import retrofit.client.Response;
 
 public abstract class AbstractSdkProfile implements SdkProfile {
-
+    private final String STATE = "state";
```

Comment:
I think the repo style is to use the string literal and do not defined constants and you should adhere to that.